### PR TITLE
this fixes #92 but this is not a definitive solution.

### LINKED ISSUE #92: Make entering a group a seamless experience to the user

### PR DESCRIPTION
**Pull request by Viterbo**
# Description
The final solution will include the registration and voting actions in the same transaction. Somehow this is not working because this gets an error from the smart contract described [here](https://github.com/telosnetwork/app-telos-native/issues/92#issuecomment-1255382749).

What this commit brings is a two-step solution where the user first needs to register and then in a separate transaction can vote.

Fixes #92 partially

image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,3 +32,4 @@ yarn-error.log*
 *.ntvs*
 *.njsproj
 *.sln
+Notas.txt
```
Comment (karynemayer) [inline .gitignore:35]: I think we should delete this, since it seems more like a personal file

File: src/components/ActionBar.vue [hunk 2]
```diff
@@ -27,7 +27,9 @@ export default {
       isFilterMenu320Open: false,
       typeGroup: [],
       typeOptions: [
+        { label: "Election", value: "election" },
```
Comment (karynemayer) [inline src/components/ActionBar.vue:30]: We have had them removed on purpose here  #77

File: src/components/ActionBar.vue [hunk 3]
```diff
@@ -620,16 +622,16 @@ div.bar-filter-wrapper
...
-        <!--btn.create-poll-btn.btn-320(-->
-          <!--iconRight-->
-          <!--:labelText="$t('pages.trails.ballots.actionBar.btnCreatePoll')"-->
-          <!--primary-->
-          <!--btnWidth='155'-->
-          <!--fontSize='16'-->
-          <!--padding='0'-->
-          <!--@clickBtn="isAuthenticated ? openBallotForm() : openNotice()"-->
-        <!--)-->
+      div.btn-320-wrapper.row.items-center.scroll-anim
```
Comment (karynemayer) [inline src/components/ActionBar.vue:625]: We also had this one removed on purpose #102

File: src/pages/trails/ballots/components/BallotForm.vue [hunk 4]
```diff
@@ -85,6 +85,8 @@ export default {
       return this.form.treasurySymbol?.symbol !== "VOTE" && this.isStakeable;
     },
     available() {
+      // TODO: borrame
+      console.log("BallotForm.computed.available -> this.userBalance: ", this.userBalance);
```
Comment (karynemayer) [inline src/pages/trails/ballots/components/BallotForm.vue:89]: I guess we can delete this 🤔

File: src/store/trails/mutations.js [hunk 5]
```diff
@@ -53,6 +53,7 @@ export const setBallot = (state, ballot) => {
 
 export const setUserTreasuries = (state, treasuries) => {
   state.userTreasuries = treasuries;
+  console.log("Trails.mutations.setUserTreasuries() ", treasuries.rows);
```
Comment (karynemayer) [inline src/store/trails/mutations.js:56]: I think this could also be removed

Comment (Viterbo) [reply src/store/trails/mutations.js:56]: totally

Comment (Viterbo) [reply src/store/trails/mutations.js]: fixed here [f6280a2](https://github.com/telosnetwork/app-telos-native/pull/125/commits/f6280a25e41c0e182dd94fbddd14dbb7d0865993)

File: src/pages/trails/ballots/components/BallotForm.vue (hunk 4 above)
Comment (Viterbo) [reply src/pages/trails/ballots/components/BallotForm.vue]: [f6280a2](https://github.com/telosnetwork/app-telos-native/pull/125/commits/f6280a25e41c0e182dd94fbddd14dbb7d0865993)

File: src/components/ActionBar.vue (hunk 2 above)
Comment (Viterbo) [reply src/components/ActionBar.vue]: ok. i fixed this in this commit 7307b8736b3bdc12591e85f35dc69881886193f2

File: .gitignore (hunk 1 above)
Comment (Viterbo) [reply .gitignore:35]: Yes, it is a personal file with notes. I put it in git ignore list just not to bother other devs. 
Should I take it away?

File: src/components/ActionBar.vue (hunk 3 above)
Comment (Viterbo) [reply src/components/ActionBar.vue:623]: I did rebase the branch with develop after you merged the [PR#123](https://github.com/telosnetwork/app-telos-native/pull/123)

File: .gitignore (hunk 1 above)
Comment (donnyquixotic) [reply .gitignore:35]: yea

Comment (Viterbo) [reply .gitignore]: just did in [ca64c76](https://github.com/telosnetwork/app-telos-native/commit/ca64c7610bda65f3d92bed82d50f6256c710131b)

**Review by donnyquixotic — APPROVED** [review]

**Review by karynemayer — APPROVED** [review]

**Pull request merged**